panoptes_client/workflow.py

```diff
@@ -179,6 +181,98 @@ def subject_workflow_statuses(self, subject_set_id):
         for status in SubjectWorkflowStatus.where(subject_ids=subject_ids, workflow_id=self.id):
             yield status
 
+    """ CAESAR METHODS """
+
+    def add_to_caesar(self):
+        caesar = Caesar()
+        payload = {
+            'workflow': {
+                'id': self.id
+            }
+        }
+        return caesar.http_post(self._api_slug, json=payload)
```

Comment:
Not blocking - can this be written as ? seems to reduce the boiler plate
```suggestion
        return Caesar().http_post(self._api_slug, json=payload)
```

Alternatively - could you add a private method or instance attribute that returns the caesar instance** vs creating one each time inline? 

**is re-using one Caesar instance on the class / via a method thread safe? If not sure then have a private method that always returns a new Caesar instance